pkg/storage/tree/collapsed.go

```diff
@@ -0,0 +1,22 @@
+package tree
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+)
+
+func (t *Tree) Collapsed() string {
```

Comment:
`Collapsed` seems quite close to `String` method. Would it make sense to reuse/adapt `String` instead?

Reply:
Yes, I think it makes sense to have a single version and adapt unit tests if needed, as you did :+1: